app/api/routers/breeze_buddy/analytics/__init__.py

```
@@ -107,3 +115,67 @@ async def get_analytics(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail=f"Error processing analytics request: {str(e)}",
         )
+
+
+@router.get("/recording/{call_sid}")
+async def get_call_recording(
+    call_sid: str,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)